### PR TITLE
fix(actuator): diasble actuator by default

### LINKED ISSUE #3981: [Bug]: The actualtor occupies port 8089, which conflicts with the oms port and causes startup failure.
### ODC version

ODC4.3.2-bp2

### OB version

No related

### What happened?

The actualtor occupies port 8089, which conflicts with the oms port and causes startup failure.

### What did you expect to happen?

The private cloud version cancels the cloud monitoring related features.

### How can we reproduce it (as minimally and precisely as possible)?

--

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by ungreat**
#### Which issue(s) this PR fixes:

Fixes #3981

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-server/src/main/resources/config/application.yml [hunk 1]
```diff
@@ -75,8 +75,14 @@ management:
     web:
       exposure:
         include: "business,application"
+    enabled-by-default: ${odc.system.monitor.actuator.enabled:false}
```
Comment (MarkPotato777) [inline server/odc-server/src/main/resources/config/application.yml:78]: The config in `application.yml` could be directly insert into `config_system_configuration`. For example, `INSERT INTO config_system_configuration(`key`, `value`, `description`) VALUES('management.endpoints.enabled-by-default', 'false', 'xxxxxx') ON DUPLICATE KEY UPDATE `id`=`id`;`

File: server/odc-server/src/main/resources/config/application.yml [hunk 2]
```diff
@@ -75,8 +75,14 @@ management:
...
+      prometheus:
+        enabled: ${odc.system.monitor.actuator.enabled:false}
+
   endpoint:
     prometheus:
-      enabled: true
+      enabled: ${odc.system.monitor.actuator.enabled:false}
   server:
-    port: 8089
+    port: ${odc.system.monitor.actuator.port:-1}
```
Comment (MarkPotato777) [inline server/odc-server/src/main/resources/config/application.yml:88]: Is -1 a illegal port value?

File: server/odc-server/src/main/resources/config/application.yml (hunk 1 above)
Comment (ungreat) [reply server/odc-server/src/main/resources/config/application.yml:78]: in order to use same key to control

File: server/odc-server/src/main/resources/config/application.yml (hunk 2 above)
Comment (ungreat) [reply server/odc-server/src/main/resources/config/application.yml]: Illegal port value, removed default value. Insert value if need to

**Review by MarkPotato777 — APPROVED** [review]
LGTM

**Pull request merged**